plugins/gatsby-theme-iterative-docs/src/components/Documentation/Markdown/index.tsx

```diff
@@ -58,11 +60,17 @@ const Details: React.FC<{ slugger: GithubSlugger }> = ({
         ? cur?.props?.children?.toString()
         : '')
   }, '')
-
-  let slug = slugger.slug(title)
-  slug = slug.replaceAll('️', '').replaceAll('ℹ', '')
-  const id = slug.replace(/(^\-+|\-+$)/g, '')
-
+  try {
+    id = id
+      ? useMemo(() => slugger.slug(id), [id])
+      : useMemo(() => slugger.slug(title), [title])
+  } catch (error) {
+    throw new Error(
+      `Duplicate ${id ? `id: "${id}"` : `title: "${title}"`} \n in page: ${
+        location.pathname
+      }`
```

Comment:
I think just the id would be fine such that the `try/catch` can be removed and the underlying Slugger's error message would be what's shown; the id is what's really what is being checked and it's not too difficult to figure out which title created the offending id.

Reply:
Yes, the error generated from the Slugger should be enough and I agree they can be figured out, but here I try to be more descriptive with `id` or `title` and `current-pathname` on the error description so that the #docs team can find it easy.